src/composables/useTable.ts

```diff
@@ -165,11 +171,101 @@ export const useTable = () => {
     }
   };
 
+  const sortByTeam = (
+    rows: readonly TableRow[],
+    sortBy: string,
+    descending: boolean,
+  ): readonly TableRow[] => {
+    const data = [...rows];
+
+    if (sortBy) {
+      data.sort((a, b) => {
+        const aVal = a[sortBy] ? a[sortBy] : '';
+        const bVal = b[sortBy] ? b[sortBy] : '';
+        if (!aVal || !bVal) return 0;
+        if (aVal < bVal) return descending ? 1 : -1;
+        if (aVal > bVal) return descending ? -1 : 1;
+        return 0;
+      });
+    }
+
+    data.sort((a, b) => {
+      if (!a.team || !b.team) return 0;
```

Comment:
Same as my previous comment https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/444/files#r1645663543.

Reply:
Updated using the if - else if structure.